runtime/flashbox/src/lib.rs

```diff
@@ -761,6 +761,8 @@ impl RegistrarHooks for FlashboxRegistrarHooks {
         // Remove bootnodes from pallet_data_preservers
         DataPreservers::para_deregistered(para_id);
 
+        ServicesPayment::on_deregister(para_id);
```

Comment:
The call to `ServicesPayment::set_credits` from above can now be moved to inside this function

Reply:
yep true